src/cdf/types.ts

```diff
@@ -238,5 +239,6 @@ export interface RelationshipsFilter {
     min: number;
   };
   sourceExternalIds?: string[];
-  targetTypes?: string[];
+  targetTypes?: boolean;
```

Comment:
As my comment above, The variable name `targetTypes` seems vague considering that its a `boolean`, should be better off if renamed to something with an `is<VarName>` like `isTypeTimeseries`